internal/acceptance/notification_destination_test.go

```diff
@@ -0,0 +1,44 @@
+package acceptance
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccNDEmail(t *testing.T) {
+	display_name := "Email Notification Destination - " + qa.RandomName()
+	workspaceLevel(t, step{
+		Template: `
+		resource "databricks_notification_destination" "this" {
+			display_name = "` + display_name + `"
+			config {
+				email {
+					addresses = ["` + qa.RandomEmail() + `"]
+				}
+			}
```

Comment:
It would be really nice to add test cases for all supported types - to catch configuration drift, etc.